CONTRIBUTING.md

```diff
@@ -22,4 +22,4 @@ $ cp pre-commit .git/hooks/pre-commit
 
 ## Adding a new Ruby version
 
-Add the new version in `ruby-builder-versions.js` and update the README.
+Add the new version in `ruby-builder-versions.js` and in `dist/index.js`. Then update the "Supported Versions" section of the README as needed.
```

Comment:
`dist/index.js`` shouldn't be edited manually, instead it should be regenerated as documented in the above sections.
Could you adapt the wording to make that clear?